argilla/src/argilla/settings/_io/_hub.py

```diff
@@ -223,24 +229,24 @@ def _define_settings_from_features(
         else:
             warnings.warn(f"Feature '{name}' has an unsupported type. Skipping. Feature type: {feature_type}")
 
-    mapping = {
-        key: value[0] if isinstance(value, list) and len(value) == 1 else tuple(value) for key, value in mapping.items()
-    }
-
     if not questions:
-        questions.append(TextQuestion(name="comment", required=True))
+        questions.append(LabelQuestion(name="quality", required=True, labels=["good", "bad"]))
+        warnings.warn(
```

Comment:
I made the message closer to the snippet and dropped the warning
<img width="758" alt="Screenshot 2024-09-24 at 16 01 33" src="https://github.com/user-attachments/assets/35616f26-884c-483a-99e8-82dcc22fa963">

If I change the order of the options in the snippet, I will need to define `rg.Settings` twice. It might become a bit complex?